Detectors/DCS/testWorkflow/DCSRandomDataGeneratorSpec.h

```diff
@@ -68,12 +69,20 @@ std::vector<int> generateIntegers(size_t size, int min, int max)
   * @returns a vector of DataPointCompositeObjects
   */
 std::vector<o2::dcs::DataPointCompositeObject> generate(const std::vector<HintType> hints,
-                                                        float fraction = 1.0)
+                                                        float fraction = 1.0,
+							uint64_t tfid = 0)
 {
   std::vector<o2::dcs::DataPointCompositeObject> dataPoints;
 
-  auto GenerateVisitor = [](const auto& t) {
-    return o2::dcs::generateRandomDataPoints({t.aliasPattern}, t.minValue, t.maxValue);
+  TDatime d;
```

Comment:
@aphecetche : this is to be able to change the timestamp at every map that we send, using the TF id.